Gemfile

```diff
@@ -12,4 +12,7 @@ gemspec
 
 ruby '~> 2.4'
 
+gem 'sqlite3', '~> 1.3.6'
```

Comment:
These dependencies should be in the `gemspec` file, otherwise we'll start installing sqlite into our main application.